src/it/resources/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks/InputEmptyBlocksAndCatchBlocksNoViolations.java

```diff
@@ -5,99 +5,92 @@
 
 import java.io.IOException;
 
-public class InputEmptyBlocksAndCatchBlocksNoViolations
-{
-    private void foo6() {
-        try {
-            throw new IOException();
-        } catch (IOException expected) { // This is expected
-            int k = 0;
-        }
+public class InputEmptyBlocksAndCatchBlocksNoViolations {
+  private void foo6() {
+    try {
+      throw new IOException();
+    } catch (IOException expected) { // This is expected
+      int k = 0;
     }
+  }
 
-    public void testTryCatch()
-    {
-        try {
-            int y=0;
-            int u=8;
-            int e=u-y;
-            return;
-        }
-        catch (Exception e) {
-            System.identityHashCode(e);
-            return;
-        }
-        finally
-        {
-            return;
-        }
+  public void testTryCatch() {
+    try {
+      int y = 0;
+      int u = 8;
+      int e = u - y;
+      return;
+    } catch (Exception e) {
```

Comment:
Why was this changed and does our google config agree? (Line separation removed)
Same with finally.

Reply:
we are not applying Whole google yet, we just do auto format and keep chapter related Checks only.

applying whole config will be separate iteration of updates.

Reply:
See other comment.

Reply:
fact that we do not use whole config yet:
https://github.com/checkstyle/checkstyle/blob/9ee77481f67b40edd41e84e2dccd5ce5ae7cd897/src/it/java/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks/EmptyBlocksMayBeConciseTest.java#L39-L42

Reply:
@Zopsss , please recheck this diff line, whole exception block is gone, not expected.

Reply:
> Why was this changed and does our google config agree? (Line separation removed) Same with finally.

yes our google config agrees with this change. I checked it with verifyWithWholeConfig method locally, and it is not giving error for having `catch (Exception e) {` on the same line as `}`, same with the finally.

Reply:
> @Zopsss , please recheck this diff line, whole exception block is gone, not expected.

the exception block is still there:

https://github.com/checkstyle/checkstyle/blob/6f8592f629c8a398b11eeb69d39f5f619904c8b5/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks/InputEmptyBlocksAndCatchBlocksNoViolations.java#L23-L26

Reply:
Confirmed, there is catch.
Strange, my bad.

I am fully satisfied on this review point

Reply:
confirming that our google config agrees with the formatted code:

```java
/** Some javadoc. */
public class Test {
  /** Some javadoc. */
  public void testTryCatch3() {
    try {
      int y = 0;
      int u = 8;
      int e = u - y;
    } catch (IllegalArgumentException e) {
      System.identityHashCode(e); // some comment
      return;
    } catch (IllegalStateException ex) {
      System.identityHashCode(ex);
      return;
    }
  }
}
```

```
java -jar .\checkstyle-10.17.0-all.jar -c .\google_checks.xml .\Test.java
Starting audit...
Audit done.
```

Reply:
> confirming that our google config agrees with the formatted code

This just shows that our google config does not put any violations on the "formatted" code. Confirmation requires that our google config would violate the original code and require us to change it to this new version.

Reply:
```java
/** Some javadoc. */
public class Test {
  /** Some javadoc. */
  public void testTryCatch3() {
    // formatted code
    try {
      int y = 0;
      int u = 8;
      int e = u - y;
    } catch (IllegalArgumentException e) {
      System.identityHashCode(e); // some comment
      return;
    } catch (IllegalStateException ex) {
      System.identityHashCode(ex);
      return;
    }

    // original code
    try {
        int y=0;
        int u=8;
        int e=u-y;
        return;
    }
    catch (Exception e) {
        System.identityHashCode(e);
        return;
    }
    finally
    {
        return;
    }
  }
}
```

```
$ java -jar .\checkstyle-10.17.0-all.jar -c .\google_checks.xml .\Test.java                       
Starting audit...
[WARN] \Test.java:18:9: 'try' child has incorrect indentation level 8, expected level should be 6. [Indentation]
[WARN] \Test.java:18:14: WhitespaceAround: '=' is not followed by whitespace. Empty blocks                may only be represented as {} when not part of a multi-block statement (4.1.3) [WhitespaceAround]
[WARN] \Test.java:18:14: WhitespaceAround: '=' is not preceded with whitespace. [WhitespaceAround]
[WARN] \Test.java:19:9: 'try' child has incorrect indentation level 8, expected level should be 6. [Indentation]
[WARN] \Test.java:19:14: WhitespaceAround: '=' is not followed by whitespace. Empty blocks                may only be represented as {} when not part of a multi-block statement (4.1.3) [WhitespaceAround]
[WARN] \Test.java:19:14: WhitespaceAround: '=' is not preceded with whitespace. [WhitespaceAround]
[WARN] \Test.java:20:9: 'try' child has incorrect indentation level 8, expected level should be 6. [Indentation]
[WARN] \Test.java:20:14: WhitespaceAround: '=' is not followed by whitespace. Empty blocks                may only be represented as {} when not part of a multi-block statement (4.1.3) [WhitespaceAround]
[WARN] \Test.java:20:14: WhitespaceAround: '=' is not preceded with whitespace. [WhitespaceAround]
[WARN] \Test.java:20:16: WhitespaceAround: '-' is not followed by whitespace. Empty blocks                may only be represented as {} when not part of a multi-block statement (4.1.3) [WhitespaceAround]
[WARN] \Test.java:20:16: WhitespaceAround: '-' is not preceded with whitespace. [WhitespaceAround]
[WARN] \Test.java:21:9: 'try' child has incorrect indentation level 8, expected level should be 6. [Indentation]
[WARN] \Test.java:22:5: '}' at column 5 should be on the same line as the next part of a multi-block statement (one that directly contains multiple blocks: if/else-if/else, do/while or try/catch/finally). [RightCurlySame]
[WARN] \Test.java:24:9: 'catch' child has incorrect indentation level 8, expected level should be 6. [Indentation]
[WARN] \Test.java:25:9: 'catch' child has incorrect indentation level 8, expected level should be 6. [Indentation]
[WARN] \Test.java:26:5: '}' at column 5 should be on the same line as the next part of a multi-block statement (one that directly contains multiple blocks: if/else-if/else, do/while or try/catch/finally). [RightCurlySame]
[WARN] \Test.java:28:5: '{' at column 5 should be on the previous line. [LeftCurly]
[WARN] \Test.java:28:5: 'finally lcurly' has incorrect indentation level 4, expected level should be 6. [Indentation]
[WARN] \Test.java:30:5: 'finally rcurly' has incorrect indentation level 4, expected level should be 6. [Indentation]
Audit done.
```

we're agreeing with formatter

Reply:
Top portion is missing finally, excess indentation and whitespace violations is making it harder to review, but it is there, so I am good.